ads/strossle.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+const renderTo = 'strossle-widget';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function strossle(global, data) {
+  validateData(data, ['widgetid']);
+  global._strossle = global._strossle || {
+    widgetId: data['widgetid'],
+  };
+
+  createContainer(global, data);
+  loadScript(
+    global,
+    'https://widgets.sprinklecontent.com/v2/sprinkle.js',
+    () => {}
+  );
+}
+
+/**
+ * @param {!Window} global
+ * @param {string} data
+ */
+function createContainer(global, data) {
+  const d = global.document.createElement('div');
+  d.className = renderTo;
+  d.setAttribute('data-spklw-widget', data.widgetid);
+  global.document.getElementById('c').appendChild(d);
```

Comment:
So just noticed this, can we use a better identifier by any chance for selecting the element?

cc @kristoferbaxter @zhouyx

Reply:
@torch2424 yarn.lock added back (without any changes from my side), thanks for your comment above. Please confirm that, and if all good for now, let's merge it 😄

Reply:
@torch2424 
This is used to select the container here
https://github.com/ampproject/amphtml/blob/bd9c449ec893b5dd7b51a495c0a69c003394c916/3p/frame.max.html#L8-L10

I don't think there's any identifier except the `id=c`. The selector looks good to me. Thanks.

Reply:
@Serzhenka Thank you! 😄 

@zhouyx Thanks for taking a look! 😄